modules/custom-post-types/portfolios.php

```diff
@@ -65,6 +65,14 @@ function __construct() {
 		add_shortcode( 'portfolio',                                                    array( $this, 'portfolio_shortcode' ) );
 		add_shortcode( 'jetpack_portfolio',                                            array( $this, 'portfolio_shortcode' ) );
 
+		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
```

Comment:
It could be nice to update the name of the filter on .com. There are 27 instances of it being used in VIP themes. When this gets into Jetpack, having one filter would make our code simpler and allow them to have simpler development code longterm.

It'd need a shim on dotcom (`wpcom_sitemap_post_types` calling `jetpack_sitemap_post_types`).